.travis.yml

```diff
@@ -1,13 +1,28 @@
 language: node_js
+dist: xenial
 node_js:
   - "10.16.0"
-sudo: true
+sudo: false
 
 addons:
-  chrome: stable
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+cache:
+  directories:
+    - ./node_modules
+
+install:
+  - npm install

```

Comment:
npm ci

Reply:
npm ci is called per default via the VM-init-settings. install is used to generate a -lock file to cache the node_module-folder. If a new commit is pushed, travis diffs the lock and if nothings changed it will use the cached node_modules-folder

Reply:
but the lock file is already there (every dev should push an updated lock file if the package.json is changed)

Reply:
Seems like Travis is caching node_modules per default now.
See https://docs.travis-ci.com/user/languages/javascript-with-nodejs#npm-ci-support